test/itemrenderer.cpp

```diff
@@ -104,7 +104,12 @@ TEST_CASE("item_renderer::to_plain_text() produces a rendered representation "
 			"Flags: " + ITEM_FLAGS_RENDERED + '\n' +
 			"Podcast Download URL: " + ITEM_ENCLOSURE_URL + '\n' +
 			" \n" +
-			ITEM_DESCRIPTON_RENDERED + '\n';
+			"Link[1]\n" +
```

Comment:
Shouldn't this be mutually exclusive with "Podcast Download URL"? I think we should have separate tests for cases where enclosure has a podcast-like type and where it has some other type.

Reply:
I guess that would make sense, I will update this PR at a later moment.

Reply:
Makes sense, I've made the options mutually exclusive. It is handled like a podcast if the enclosure type is empty or a valid podcast type. In the other cases, it is shown in this new way.